src/utils/extractPaginationParameters.ts

```diff
@@ -0,0 +1,28 @@
+import type {
+  Request,
+} from 'express';
+import type {
+  PaginationParameters,
+} from '../types';
+
+const parseIntegerQueryParameter = (
+  req: Pick<Request, 'query'>,
+  parameterName: string,
+  defaultValue: number,
+): number => {
+  const rawValue = req.query[parameterName];
+  if (typeof rawValue === 'string') {
+    const parsedValue = parseInt(rawValue, 10);
+    return Number.isNaN(parsedValue)
```

Comment:
Should we throw an error if an invalid string is provided or should we treat it as though it wasn't passed in the first place?

Error would be more explicit, so I'm leaning towards changing this to throw a validation error.